webpack/assets/javascripts/react_app/components/notifications/toggleIcon/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default ({ hsUnreadMessages, onClick }) => {
+  const iconType = hsUnreadMessages ? 'fa-bell' : 'fa-bell-o';
+
+   return (
+   <a className="nav-item-iconic drawer-pf-trigger-icon" onClick={onClick}>
+        <span className={`fa ${iconType}`} title={__('Notifications')}>
```

Comment:
tooltips: absolutly yes.
Icon - not sure, i dont really understand the exiting implementation, and how to use it...
we should make an effort to standardize icons in a better, more friendly way